modules/installation-special-config-mirrored-disk.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-mirrored-disk_{context}"]
+= Mirroring disks during installation
+
+During {product-title} installation, you can enable mirroring of the boot disk to two or more redundant replicas on master and worker nodes. A node stays active after failure as long as one replica still functions. Mirroring support is available on x86_64 nodes booted with BIOS or UEFI and on ppc64le nodes.
+
+Mirroring does not support replacement of a failed disk. To restore the mirror to a pristine, non-degraded state, reprovision the node. For disk encryption, an alternate method of configuring disk encryption is required.
+
+[IMPORTANT]
+====
+Mirroring is available only for user-provisioned infrastructure deployments on {op-system-first} systems.
+====
+
+.Procedure
+
+To enable boot disk mirroring during {product-title} deployment:
+
+. Follow the bare metal install procedure up to the point where you created Ignition configs for your installation.
++
+For example, you should have entered the following command:
++
+[source,terminal]
+----
+$ openshift-install create ignition-configs --dir $HOME/clusterconfig
+----
++
+. Verify that the Ignition config files were created:
++
+[source,terminal]
+----
+$ ls $HOME/clusterconfig/
+----
++
+[source,terminal]
+.Example output
+----
+auth  bootstrap.ign  master.ign  metadata.json  worker.ign
+----
+
+. Create a {op-system} Config (RCC) with a reference to your `master.ign` or `worker.ign` config, depending on the type of node you are configuring. The RCC must specify the devices to be used for boot disk mirroring. If you want, you can configure LUKS encryption for the root filesystem.
```

Comment:
If they want LUKS, they must specify it here and not in the machine config.  `If you want LUKS encryption on the mirrored root filesystem, configure it here.`?

Reply:
That makes more sense. To clarify further, how about "...configure it in this RCC"?

Reply:
Sounds good.